hbase-server/src/main/java/org/apache/hadoop/hbase/security/HBasePolicyProvider.java

```diff
@@ -44,8 +46,11 @@ public class HBasePolicyProvider extends PolicyProvider {
       new Service("security.client.protocol.acl",
         RegistryProtos.ClientMetaService.BlockingInterface.class),
       new Service("security.admin.protocol.acl", MasterService.BlockingInterface.class),
+      new Service("security.admin.protocol.acl", LockService.BlockingInterface.class),
```

Comment:
Is there a reason to add LockService in this PR? I suppose there's no reason not to, just wondering (i presume LockService has been working as-is for whatever reason)

Reply:
I added a UT in this PR to find out the inconsistency between HBasePolicyProvider, SecurityInfo and the services deployed in our MasterRpcServices and RSRpcServices, and then found out that LockService is missing, so it should have been broken for a long time.

I searched the code base, had not found any references to this service, maybe it is not exposed through our normal client API, but since it is exposed as an rpc service, we should fix it anyway.

Reply:
Sounds good, thanks